.drone.yml

```diff
@@ -59,23 +58,6 @@ steps:
     instance:
     - drone-pr.rancher.io
 
-
```

Comment:
Why it would fail? I think it is better that we still run upstream tests on our forks in case we introduce regression on our commits.

Reply:
https://github.com/rancher/ingress-nginx/pull/71#issue-1031892747 explained here